### PR TITLE
Footnote on cross-library dependencies pinning date

### PR DESCRIPTION
**Pull request by pentschev**
As was discussed today regarding Dask pinning, it was decided it should happen during the second code-freeze date. Adding a footnote explaining that.

### CODE REVIEW

**Review by ajschmidt8 — APPROVED** [review]
LGTM. Let's wait to double check this on the preview environment that gets deployed

**Comment by jakirkham** [comment]
cc @galipremsagar

**Review by galipremsagar — APPROVED** [review]

**Comment by pentschev** [comment]
Formatting looks ok to me, see https://deploy-preview-309--docs-rapids-ai.netlify.app/maintainers#fn:1 .

**Comment by galipremsagar** [comment]
> Formatting looks ok to me, see https://deploy-preview-309--docs-rapids-ai.netlify.app/maintainers#fn:1 .

Yup. @ajschmidt8 could you also have a look at preview before we merge?

**Comment by jakirkham** [comment]
Including a screenshot:

Screen Shot 2022-11-16 at 1 06 54 PM

**Comment by jakirkham** [comment]
Do we know what the little arrows after the footnote are for? Tried clicking them and nothing happened. Though could be a browser thing (using Safari)

File: maintainers/maintainers.md
Comment (galipremsagar) [inline maintainers/maintainers.md:52]:
```suggestion
[^1]: Cross-library dependencies are pinned during this time-frame. For example: Dask and Distributed.

```

Comment (galipremsagar) [reply maintainers/maintainers.md:52]: @jakirkham Just trying to see if this can go away by adding a newline later.

Comment (galipremsagar) [reply maintainers/maintainers.md:52]: Ignore.

**Comment by galipremsagar** [comment]
> Do we know what the little arrows after the footnote are for? Tried clicking them and nothing happened. Though could be a browser thing (using Safari)

That seems to be the expected behavior of markdown, for example see:
Screen Shot 2022-11-16 at 3 16 53 PM
Screen Shot 2022-11-16 at 3 16 48 PM

**Comment by pentschev** [comment]
Yeah, the arrows are [supposed to be a reverse link](https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/), but I guess it only matters if the page is long enough.

**Comment by pentschev** [comment]
To cheat that, you can resize your browser window vertically so that it's very short and then clicking the first arrow will jump to the first reference.

**Comment by jakirkham** [comment]
Thanks all! 🙏

**Pull request merged**
